### PR TITLE
Make Cucumber features runnable from the command line

### PR DESCRIPTION
**Pull request by cbbayburt**
This change moves environment setup to `.bashrc` for the test controller so that the individual cucumber features can be runnable from the command line as below:

```
cucumber features/my_feature.feature
```

### CODE REVIEW

**Comment by MalloZup** [comment]
lgtm for me

File: salt/controller/init.sls [hunk 1]
```diff
@@ -86,3 +86,12 @@ cucumber_run_script:
     - user: root
     - group: root
     - mode: 755
+
+env-vars:
```
Comment (moio) [inline salt/controller/init.sls]: Please use underscore_separation for Salt IDs. Moreover keep in mind those should be unique, so eg. `testsuite_environment_variables`

Comment (MalloZup) [inline salt/controller/init.sls]: good catch

**Comment by cbbayburt** [comment]
@moio, I pushed the changes. Thanks for the review!

**Comment by mbologna** [comment]
Tested, this is useful. Thanks!

**Comment by moio** [comment]
Thanks @cbbayburt, sorry for merging this late! For some reason I missed GitHub's notification.

Very appreciated!

**Pull request merged**